### PR TITLE
Add the status of the translation in the discussion

### LINKED ISSUE #103: Add the status of the translation in the discussion
## What problem does this address?

When you are in the discussion page for an original, you don't get the translation status of the translation related with each comment. 

image

## What is your proposed solution?

Add the translation status related with the comment.

### PR DESCRIPTION
**Pull request by amieiro**
## Problem

When you are in the discussion page for an original, you don't get the translation status of the translation related with each comment. 

Fixes #103. 

image

## Solution

This PR adds the translation status related with the comment.

image

image

## Testing instructions

For an original string:
* Suggest a translation.
* Reject this suggestion with comment.
* Suggest another different translation.
* Accept this suggestion with comment.
* Go to the discussion page for this original. You will see the translation status of the string related with the comment.

### CODE REVIEW

**Comment by trymebytes** [comment]
I tested this and the status highlighted in the screenshot below was initially `rejected` but when I  later approved (and added feedback) the same translation, the status changed to `current`. Both comments now have the same translation status.
Screenshot 2022-08-03 at 09 48 48

**Comment by akirk** [comment]
So this means we'd have to store the translation status in the comment meta if we want to make sure it reflects the status at the time, right?

**Comment by trymebytes** [comment]
Yes we will need to store the translation status.

File: helpers-assets/templates/translation-discussion-comments.php [hunk 1]
```diff
@@ -167,6 +167,5 @@ function () use ( $locale_slug, $options ) {
 		/* translators: Log in URL. */
 		echo sprintf( __( 'You have to be <a href="%s">logged in</a> to comment.' ), esc_html( wp_login_url() ) );
 	}
-
```
Comment (akirk) [inline helpers-assets/templates/translation-discussion-comments.php:170]: Stray modification?

**Comment by akirk** [comment]
@amieiro this PR is yours but @trymebytes has been completing it, so could you please review? Thanks!

**Comment by amieiro** [comment]
I have tested it in my local environment and I think it is ready to be merged.
 
image

**Review by akirk — APPROVED** [review]
Approved by @amieiro who accidentally also owns the PR.

**Pull request merged**